WolframLanguage.sublime-syntax

```diff
@@ -90,14 +106,19 @@ contexts:
         pop: true
 
   literals:
-    - match: (?:\d+\.?|\.\d)\d*\`{0,2}
+    # numbers, with base, scientific notation, precision or accuracy
+    - match: ([0-9]+\^\^)?{{number}}(\`\`?({{number}})?)?(\*\^{{number}})?
```

Comment:
The first `{{number}}` can contain letters to represent digits in bases higher than 10.
I was working on this syntax but we can do it here as well. This would be my proposal
```suggestion
    - match: |-
        (?x)
        (?:(\d+)(\^\^))
        ((?:[$[:alnum:]]+\.?|\.[$[:alnum:]])[$[:alnum:]]*)
        (?:(\`\`?)({{number}})?)?
        (?:(\*\^)([+-]?\d+))?
      captures:
        1: constant.numeric.base.wolfram
        2: keyword.operator.base.wolfram
        3: constant.numeric.wolfram
        4: keyword.operator.precision.wolfram
        5: constant.numeric.precision.wolfram
        6: keyword.operator.exponent.wolfram
        7: constant.numeric.exponent.wolfram
    - match: |-
        (?x)
        ({{number}})
        (?:(\`\`?)({{number}})?)?
        (?:(\*\^)([+-]?\d+))?
      captures:
        1: constant.numeric.wolfram
        2: keyword.operator.precision.wolfram
        3: constant.numeric.precision.wolfram
        4: keyword.operator.exponent.wolfram
        5: constant.numeric.exponent.wolfram
```
I know it does not distinguish between precision and accuracy but I did not want to explode the pattern even more.

Reply:
We can distinguish between precision and accuracy with a little more work.

Reply:
And exponents can have a decimal point.

Reply:
> ((?:[$[:alnum:]]+\.?|\.[$[:alnum:]])[$[:alnum:]]*)

What is `$` for?

Reply:
Also, I don't like the idea to treat `^^` and `*^` as "operators". If they were operators, they should have precedence and expressions like `8 ^^ 10` should be right. But they were not.